app/models/organization_wizzard.rb

```diff
@@ -0,0 +1,31 @@
+class OrganizationWizzard
+  include Rails.application.routes.url_helpers
+
+  def initialize(organization)
+    @organization = organization
+  end
+
+  def ready?
+    have_account? && have_categories?
+  end
+
+  def not_ready?
```

Comment:
@dissident it's diffenetly ruby way :). Not ruby way is to use `if !ready?` instead `unless ready?`